build_tools/windows/windows_testing_downloader.ps1

```diff
@@ -1,270 +1,270 @@
+        "scipy" = "http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/scipy-0.14.0-cp27-none-win32.whl"
+        "get-pip" = "https://bootstrap.pypa.io/get-pip.py"
+    }
+    return $urls    
+}
+
+function Python34URLs {
+    # Function returns a dictionary of packages to download for Python 3.4.
+    $urls = @{
+        "python" = "https://www.python.org/ftp/python/3.4.1/python-3.4.1.msi"
+        "numpy" = "http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/numpy-1.8.1-cp34-none-win32.whl"
+        "scipy" = "http://28daf2247a33ed269873-7b1aad3fab3cc330e1fd9d109892382a.r6.cf2.rackcdn.com/scipy-0.14.0-cp34-none-win32.whl"
+    }
+    return $urls    
+}
+
+function GitURLs {
+    # Function returns a dictionary of packages to download for Git
+    $urls = @{
+        "git" = "https://github.com/msysgit/msysgit/releases/download/Git-1.9.4-preview20140611/Git-1.9.4-preview20140611.exe"
+    }
+    return $urls    
+}
+
+function main {
+    $versions = @{
+        "2.7" = Python27URLs
+        "3.4" = Python34URLs
+    }
+
+    if ($nogit -eq "False") {
+        Write-Host "Downloading and installing Gitbash"
+        $urls = GitURLs
+        DownloadPackages $urls ""
+        InstallGit ".exe"
+    }
+
+    if (($python -eq "None")) {
+        Write-Host "Installing all supported python versions"
+        Write-Host "Current versions supported are:"
+        ForEach ($key in $versions.Keys) {
+            Write-Host $key
+            $all_python += @($key)
+        }
+    } elseif(!($versions.ContainsKey($python))) {
+        Write-Host "Python version not recognized!"
+        Write-Host "Pass python version with -python"
+        Write-Host "Current versions supported are:"
+        ForEach ($key in $versions.Keys) {
+            Write-Host $key
+        }
+        return
+    } else {
+        $all_python += @($python)
+    }
+    ForEach ($py in $all_python) {
+        Write-Host "Installing Python" $py
+        DisableInternetExplorerESC
+        $pystring = $py -replace "\."
+        $pystring = "_py" + $pystring
+        $package_dict = $versions[$py]
+
+        # This will download the whl packages as well which is
+        # clunky but makes configuration simpler.
+        DownloadPackages $package_dict $pystring
+        UpdatePaths $py
+        InstallPython $pystring
+        ReadAndUpdateFromRegistry
+        if ($package_dict.ContainsKey("get-pip")) {
+           InstallPip $pystring $py
+        } else {
+           EnsurePip $py 
+        }
+        InstallNose $py
+        InstallWheel $py
+
+        # The installers below here use wheel packages.
+        # Wheels were created from CGohlke's installers with
+        # wheel convert <exefile>
+        # These are hosted in Rackspace Cloud Files.
+        InstallNumpy $package_dict $py
+        InstallScipy $package_dict $py
+    }
+    return
+}
+
+main
```

Comment:
Hmmm interesting why this didn't show up as a git move :/ Did you do the remove / add in separate commits? Would you mind squashing your commits please?